includes/subscriptions.php

```diff
@@ -335,13 +335,39 @@ function send_notifications( $post ) {
 				$update_subscriptions = true;
 
 				wp_delete_post( $subscription_id, true );
+
+				/**
+				 * Fires after Distributor pushes a post subscription
+				 *
+				 * @since TBD
+				 * @hook dt_after_set_meta
+				 *
+				 * @param {int}   $post_id         Post ID
+				 * @param {int}   $subscription_id Subscription ID
+				 * @param {array} $request         Request array
+				 */
+				do_action( 'dt_subscription_after_post_push', $post_id, $subscription_id, $request );
```

Comment:
Since the status is currently failure at this point, should we consider tweaking the filter name to reflect this?